api.py

```diff
@@ -109,53 +95,3 @@ def dispatcher(router, request, response):
 app.router.set_dispatcher(dispatcher)
-
-
-if __name__ == '__main__':
-    import pymongo
-    import argparse
-    import paste.httpserver
-
-    arg_parser = argparse.ArgumentParser()
-    arg_parser.add_argument('--host', default='127.0.0.1', help='IP address to bind to [127.0.0.1]')
-    arg_parser.add_argument('--port', default='8080', help='TCP port to listen on [8080]')
-    arg_parser.add_argument('--db_uri', help='SciTran DB URI', default='mongodb://localhost/scitran')
-    arg_parser.add_argument('--data_path', help='path to storage area', required=True)
-    arg_parser.add_argument('--apps_path', help='path to apps storage')
-    arg_parser.add_argument('--log_level', help='log level [info]', default='info')
-    arg_parser.add_argument('--ssl_cert', help='path to SSL certificate file, containing private key and certificate chain', required=True)
-    arg_parser.add_argument('--site_id', help='site ID for Scitran Central [local]', default='local')
-    arg_parser.add_argument('--oauth2_id_endpoint', help='OAuth2 provider ID endpoint', default='https://www.googleapis.com/plus/v1/people/me/openIdConnect')
-    arg_parser.add_argument('--demo', help='enable automatic user creation', action='store_true', default=False)
-    arg_parser.add_argument('--insecure', help='allow user info as urlencoded param', action='store_true', default=False)
-    args = arg_parser.parse_args()
-
-    args.quarantine_path = os.path.join(args.data_path, 'quarantine')
-    args.upload_path = os.path.join(args.data_path, 'upload')
-    app.config = vars(args)
-
-    logging.getLogger('paste.httpserver').setLevel(logging.WARNING) # silence paste logging
-    log.setLevel(getattr(logging, args.log_level.upper()))
-
-    if not app.config['ssl_cert']:
-        log.warning('SSL certificate not specified -> SciTran Central functionality disabled')
-    if app.config['site_id'] == 'local':
-        log.warning('site_id not configured -> SciTran Central functionality disabled')
-
-    if not os.path.exists(app.config['data_path']):
-        os.makedirs(app.config['data_path'])
-    if not os.path.exists(app.config['quarantine_path']):
-        os.makedirs(app.config['quarantine_path'])
-    if not os.path.exists(app.config['upload_path']):
-        os.makedirs(app.config['upload_path'])
-    if not app.config['apps_path']:
-        log.warning('apps_path is not defined -> apps functionality disabled')
-    elif not os.path.exists(app.config['apps_path']):
-        os.makedirs(app.config['apps_path'])
-
-    db_client = pymongo.MongoReplicaSetClient(args.db_uri) if 'replicaSet' in args.db_uri else pymongo.MongoClient(args.db_uri)
-    app.db = db_client.get_default_database()
-    app.db.sites.update({'_id': args.site_id}, {'_id': args.site_id, 'name': 'Local'}, upsert=True)
-
-    app.debug = True # send stack trace for uncaught exceptions to client
-    paste.httpserver.serve(app, host=args.host, port=args.port, ssl_pem=args.ssl_cert)
```

Comment:
Is this remove-only diff for `api.py` only remove parsing redundancy with `api.wsgi`? I assume this was because of folks using multiple ways to launch the app?